Content.Shared/Disposal/SharedDisposalUnitSystem.cs

```diff
@@ -14,11 +14,6 @@ public override void Update(float frameTime)
             {
                 comp.Update(frameTime);
             }
-
-            foreach (var comp in ComponentManager.EntityQuery<SharedDisposalMailingUnitComponent>(true))
-            {
-                comp.Update(frameTime);
-            }
```

Comment:
since you're removing DisposalMailingUnit make sure to check all references to it (i.e. in IgnoredComponents or in YAML), not sure if there are references there as well

Reply:
I couldn't find any references to DisposalMailingUnit